aws-observability-terraform/source-module/main_iam_role.tf

```diff
@@ -38,16 +38,16 @@ resource "aws_iam_role_policy_attachment" "cloudtrail_policy_attach" {
 
 # Sumo Logic ELB Source Policy Attachment
 resource "aws_iam_policy" "elb_policy" {
```

Comment:
please check the policy is applied to elb and classic lb if buckets are different.